packages/components/src/components/bal-navigation/bal-main-mobile/bal-main-mobile.tsx

```diff
@@ -0,0 +1,17 @@
+import { Component, h, ComponentInterface, Host } from '@stencil/core'
+import { BEM } from '../../../utils/bem'
+
+@Component({
+  tag: 'bal-main-mobile',
+})
+export class MainMobile implements ComponentInterface {
+  render() {
+    const mainEl = BEM.block('nav').element('mainmobile')
+
+    return (
+      <Host class={{ ...mainEl.class(), 'is-hidden-desktop': true }}>
```

Comment:
Please do not use the class helpers, those should only be used outside the DS. do it with BEM thank you

Reply:
as discussed, unnecessary component deleted